### PR TITLE
django admin: custom index template

### PR DESCRIPTION
**Pull request by domdinicola**
This PR introduces a custom index template:
- highlighting with a different color the tenant apps
- disable the tenant apps when in the public schema (avoiding exceptions)

### CODE REVIEW

**Comment by tomturner** [comment]
Thanks for this What w good idea. I will give this a look later and merge in

File: django_tenants/templates/admin/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+{% block content %}
+    <div id="content-main">
+        {% if app_list %}
+            {% for app in app_list %}
+                {% is_tenant_app_and_public_schema app as is_tenant_app_and_public %}
+                {% is_tenant_app app as is_tenant_app %}
+                <div class="app-{{ app.app_label }} module">
+                    <table id="change-history">
+                        {% if is_tenant_app_and_public %}
```
Comment (domdinicola) [inline django_tenants/templates/admin/index.html]: Hide the models when in public schema and the app is a tenant app

File: django_tenants/templates/admin/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                            <div class="module">
+                                <caption {% if is_tenant_app %}style="background-color:#f7d904"{% endif %}>
```
Comment (domdinicola) [inline django_tenants/templates/admin/index.html]: Change color of the app if it's a tenant app

File: django_tenants/templates/admin/index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                                    <div class="section" title="{% blocktrans with name=app.name %}Models in the {{ name }} application{% endblocktrans %}">{{ app.name }}</div>
+                                    <tr>
+                                        <td scope="row">{% trans "Not available for Global schema" %}</td>
+                                    </tr>
+                                </caption>
+                            </div>
+                        {% else %}
+                            <caption {% if is_tenant_app %}style="background-color:#f7d904"{% endif %} >
```
Comment (domdinicola) [inline django_tenants/templates/admin/index.html]: Change color of the app if it's a tenant app

**Comment by tomturner** [comment]
@domdinicola thanks mereged now

**Pull request merged**